src/platform/qt/CMakeLists.txt

```diff
@@ -22,13 +22,13 @@ if(BUILD_SDL)
 	if(SDL2_FOUND)
 		link_directories(${SDL2_LIBDIR})
 	endif()
-	list(APPEND PLATFORM_LIBRARY ${SDL_LIBRARY} ${SDLMAIN_LIBRARY})
+	list(APPEND PLATFORM_LIBRARY ${SDL_LIBRARY})
```

Comment:
Oh yeah, I should probably break these changes out into another PR. Without them, SDL + Qt build fails on MSVC.